lib/read-stream.js

```diff
@@ -143,15 +156,17 @@ class ReadStream extends stream.Readable {
       this._readBlock()
     }
 
-  }
+  },
 
   destroy() {
     fs.close( this.fd, ( error ) => {
       this.fd = null
       stream.Readable.prototype.destroy.call( this )
     })
-  }
+  },
 
 }
 
+inherit( ReadStream, stream.Readable )
```

Comment:
> I don't mean any disrespect, but I wonder if there's a more common / popular / well known module that does something similar?

Haha, no worries – I totally understand. I thought so as well, but even [strolling through node modules](https://npms.io/search?q=inherit) now, nothing useful comes up. The first one contains a bunch of sugar for a few things, including mixins and "special" properties like `__constructor` (which isn't necessarily bad, just not needed and might clutter objects), the second one is node core's `util.inherit()` as a module (which is the one in question to avoid), and then that's already pretty much it.

Either way, bloodline doesn't look very trustworthy as it is at the moment.
I'll should add tests and things to it, so you (and others) don't have to take my word for it ;)